packages/lighthouse-service/src/lighthouse-db-manager/lighthouse-db-manager.ts

```diff
@@ -450,6 +450,37 @@ class LighthouseDbManager {
     stat.score = leaderBoardScores[`${projectId}:${branch}`];
     return stat;
   }
+
+  async exportLHReportByProject(projectId: string, branch: string) {
+    const lhReport = await sequelize.query(
+      `
```

Comment:
Seperate this into another PR

Reply:
Please don't join backend and frontend changes in one PR. Split this into two please